transports/ssl/src/main/resources/META-INF/mule-tls.xsd

```diff
@@ -265,4 +274,99 @@
         </xsd:attribute>
     </xsd:complexType>
 
+    <xsd:complexType name="rcWrapperElement">
+        <xsd:choice minOccurs="0" maxOccurs="1">
```

Comment:
Does this mean you could have an empty revocation-check?

Reply:
Yes, that's wrong now that we have a wrapper element.